File: Java/reverse-integer/ReverseInteger.java
public int reverse(int x) {
        String aux = "";
        int result;
        if (x < 0) {
            x *= -1;
            aux = "-";
        }
        String t = Integer.toString(x);
        for (int i = t.length() - 1; i >= 0; i--) {
            aux += t.charAt(i);
        }
        try {
            result = Integer.parseInt(aux);
            return result;
        } catch (NumberFormatException E) {
            return 0;
        }

    }

File: Kotlin/reverse-integer/ReverseInteger.kt
un reverse(x: Int): Int {
    var cadena = x.toString()
    var aux = ""
    if (cadena.get(0) == '-') {
        cadena = cadena.substring(1, cadena.length)
        aux += "-"
    }
    if (cadena.get(cadena.length - 1) == '0') {
        cadena = cadena.substring(0, cadena.length - 1)
    }
    for (i in cadena.length - 1 downTo 0)
        aux += cadena.get(i)
    try {
        var e = aux.toInt()
        println(e)
        return e
    } catch (E: NumberFormatException) {
        return 0
    }
}